Add Drain Cleaner 1.1.0 to the installation files (#9768)
strimzi/strimzi-kafka-operator: documentation/modules/drain-cleaner/proc-drain-cleaner-deploying.adoc

.Procedure

> . If you use the legacy mode activated by setting the `STRIMZI_DENY_EVICTION` environment variable to `false`, you have to configure the `PodDisruptionBudget` resource.

Reviewer: ```suggestion
. If you are using the legacy mode activated by setting the `STRIMZI_DENY_EVICTION` environment variable to `false`, you must also configure the `PodDisruptionBudget` resource.
```
